src/styles/index.scss

```diff
@@ -19,4 +19,9 @@ body {
     justify-content: center;
     min-width: 320px;
     min-height: 100vh;
-}
\ No newline at end of file
+}
+
+#root {
+    /* create a stacking context for the app */
+    isolation: isolate;
```

Comment:
Could this be in the site-wrapper `root` class, avoiding needing the `#root` id addition?